src/pages/dashboard/index.tsx

```diff
@@ -0,0 +1,244 @@
+                </div>
+              </Block>
+              <Block>
+                <div className="p-5">
+                  <div className="flex items-center">
+                    <div className="flex-shrink-0">
+                      <BookmarkSquareIcon className="text-cool-gray-400 h-6 w-6" />
+                    </div>
+                    <div className="ml-5 w-0 flex-1">
+                      <DescriptionList
+                        displayMode={DescriptionListDisplayMode.LABEL_ABOVE}
+                      >
+                        <DescriptionList.Item
+                          label={
+                            <div className="text-cool-gray-500 truncate text-sm font-medium leading-5">
+                              {t("Pipelines")}
+                            </div>
+                          }
+                        >
+                          <div className="text-cool-gray-900 text-lg font-medium leading-7">
+                            {data?.dags.totalItems}
+                          </div>
+                        </DescriptionList.Item>
+                      </DescriptionList>
+                    </div>
+                  </div>
+                </div>
+                <div className="bg-cool-gray-50 px-5 py-3">
+                  <div className="text-sm leading-5">
+                    <Link
+                      href="/pipelines"
+                      className="font-medium text-blue-600 transition duration-150 ease-in-out hover:text-blue-900"
+                    >
+                      {t("View all")}
+                    </Link>
+                  </div>
+                </div>
+              </Block>
+            </div>
+          </div>
+
+          <div className="my-12">
+            <div className="flex items-center justify-between">
+              <h3 className="ml-2 mt-2 text-lg leading-6 text-gray-900">
+                {t("Last Activity")}
+              </h3>
+            </div>
+            <div className="mt-4">
+              <Block>
+                <DataGrid data={data?.lastActivities || []}>
+                  <BaseColumn id="name" label={t("Name")}>
+                    {(item) => (
+                      <div
+                        className="truncate text-sm text-gray-900"
+                        title={item.description}
+                      >
+                        {item.description}
+                      </div>
+                    )}
+                  </BaseColumn>
+                  <BaseColumn id="status" label={t("Status")}>
+                    {(item) => <ActivityStatusBadge status={item.status} />}
+                  </BaseColumn>
+
+                  <DateColumn
+                    label={t("Date")}
+                    relative
+                    accessor="occurredAt"
+                  />
+                  <ChevronLinkColumn
+                    accessor="url"
+                    url={(value: any) => ({
+                      pathname: value || "",
```

Comment:
You graphql schema says that the url is always provided so you can remove the `||""` IMHO